app/views/home/index.html.erb

```diff
@@ -1 +1,17 @@
-<h>Hello World</h>
+<main class="home">
+  <header class="home__header"><a href="#" class="home__logo">ground rules</a></header>
+  <%= render 'svg/home-ill-cup' %>
+  <section class="home__section">
+      <h1>Created for tired people, by slightly less tired people.</h1>
+      <nav class="home__nav">
+        <ul>
+          <li><a href="">Find Your Roast</a></li>
+          <li><a href="">Master Your Machine</a></li>
+        </ul>
+      </nav>
```

Comment:
Do you know what the URLs are going to be for the Roast and machine pages? Something like `/machines`?

At least throw a `#todo` comment by those links so you don't miss filling in those hrefs